server/@types/index.d.ts

```diff
@@ -162,7 +162,6 @@ export interface DecoratedRecall extends RecallResponse, DocumentDecorations {
 }
 
 export interface PersonAndRecallResponse {
```

Comment:
The whole type can probably be discarded as you're now just receiving a `RecallResponse`

Reply:
yep, good catch :) I was trying to do things in steps! Will amend